### DISCUSSION TITLE
send OTP template

### DISCUSSION THREAD

**Discussion by asim-altayb** [netflie/laravel-notification-whatsapp#17] [General]
new version of CloudApi they add OTP template , can you add support fot it

**Comment by aalbarca** [comment]
Please @asim-altayb , can you send me any doc. or link with more information? Thanks.

**Reply by Ctrl-Mota** [reply]
https://developers.facebook.com/docs/whatsapp/business-management-api/authentication-templates/copy-code-button-authentication-templates/

type: OTP

**Comment by Ctrl-Mota** [comment]
I opened a pull request for this feature, pls could review?
https://github.com/netflie/laravel-notification-whatsapp/pull/29
